infra/src/main/java/com/redhat/service/smartevents/infra/models/processors/ProcessorDefinition.java

```diff
@@ -25,25 +25,31 @@ public class ProcessorDefinition {
     @JsonProperty("resolvedAction")
     private Action resolvedAction;
 
+    @JsonProperty("isErrorHandler")
+    private boolean isErrorHandler;
```

Comment:
I need to differentiate between a _regular_ `Processor` and one for Error Handling. Adding a flag to the `Action` (used to define an Error Handler) exposes the flag in`openapi.yaml`; here it remains encapsulated in the `Processor` definition and does not _leak_ outside of RHOSE.